l10n_co_geo/data/res_bank.xml

```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We put this file in other module called `l10n_es_partner` to associate certain logic to deduce the bank from the partner bank account.